cmd/fixtures_test/send_items_test/send_items.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ItemNames": ["Sword", "Spear"],
```

Comment:
Instead of general name Sword and Spear, can you use the item name with with cookbook name as prefix?